src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -697,4 +697,13 @@
   <data name="PlainText" xml:space="preserve">
     <value>Plain Text</value>
   </data>
+  <data name="CloseOnExitInfoBar.Message" xml:space="preserve">
+    <value>Termination behavior can be configured in advanced profile settings</value>
+  </data>
+  <data name="CloseOnExitInfoBarDismissButton.Content" xml:space="preserve">
+    <value>Don't show again</value>
+  </data>
+  <data name="KeyboardServiceWarningInfoBarDismissButton.Content" xml:space="preserve">
+    <value>Don't show again</value>
+  </data>

```

Comment:
I'd prefer that we make this one resource and bind to the one resource. That'll reduce the localization burden and make it easier for us to add new "Don't show again" buttons in the future.

Reply:
I wish we didn't have to do that. But also, having multiple resources that say the same thing is not really that bad.